doc/releases/release_0.19.rst

```diff
@@ -0,0 +1,32 @@
+Version 0.19 (Expected 1st of December 2021)
+--------------------------------------------
+Minor update introducing a few new features and fixes some smaller bugs.
+
+New Features / Enhancements
```

Comment:
Add Spline

Reply:
I stopped manually adding things that others add but don't add to this list, as this is too much work. But feel free to add things of course. We should be probably more strict on this for future releases.